package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "dependencies": {
```

Comment:
What do you guys think of adding basic libraries as dependencies like: axios, bootstrap, immutable, jquery, etc @mishuagopian @sbalay